### PR TITLE
Added make file

### PR DESCRIPTION
**Pull request by raushan2016**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= raushan2016/etcdexplore
+
+# Build manager binary
+manager:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o bin/etcdexplore lucas.go
```
Comment (ringtail) [inline Makefile]: could you change it to `lucas`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= raushan2016/etcdexplore
```
Comment (ringtail) [inline Makefile]: Pls change it to `registry.cn-hangzhou.aliyuncs.com/ringtail/lucas`

Comment (raushan2016) [reply Makefile]: done

File: Makefile (hunk 1 above)
Comment (raushan2016) [reply Makefile]: done

**Comment by ringtail** [comment]
@raushan2016 Thanks man!

**Pull request merged**